daemon/api_users.go

```diff
@@ -408,6 +408,12 @@ func getUserDetailsFromStore(theStore snapstate.StoreService, email string) (str
 		return "", nil, fmt.Errorf("cannot create user for %q: no ssh keys found", email)
 	}
 
+	// Amend information where the key came from to ensure it can
+	// be update/replaced later
+	for i, k := range v.SSHKeys {
+		v.SSHKeys[i] = fmt.Sprintf(`%s # {"origin":"store","email":%q}`, k, email)
```

Comment:
The `#` is only used to mark comments when used at the beginning of a line. It's also true that anything written after the key is treated as a comment, till the end of the line, so the `#` does not harm either. I would probably remove it, but I don't have a strong opinion.

Reply:
Woah, I had no idea about this, thank you!

Reply:
I updated this now and left the "#" mostly because tools like ssh-import-id also use the `"# ssh-import-id %s:%s"`